apps/src/templates/PaneHeader.jsx

```diff
@@ -186,7 +186,7 @@ export const PaneButton = Radium(function(props) {
       const Icon = icon.type;
       return (
         <Icon {...icon.props} style={{...iconStyle, ...icon.props.style}}>
-          {icon.children}
+          {icon.props.children}
```

Comment:
I think this was just a typo -- I looked at the PR where this icon prop was added, and do not see anywhere that children would be used. I also checked all current uses of PaneButton -- most use iconClass, but few that use icon use a node that has no children.